.bazelrc

```diff
@@ -0,0 +1,23 @@
+# 
+#  Copyright 2020 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     https://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# 
+
+# Common flags for Clang
+build:clang --repo_env=CC=clang
+
+# Common flags for Clang
+build:asan-libfuzzer --config=clang
+build:asan-libfuzzer --linkopt -fsanitize=fuzzer,address
+build:asan-libfuzzer --copt -fsanitize=fuzzer,address
```

Comment:
Maybe we can use the "=" form of the flag, to clarify it's a single flag? E.g.: `--linkopt=-fsanitize=fuzzer,address`.